Sources/SwifterSwift/UIKit/UIViewExtensions.swift

```diff
@@ -329,6 +329,20 @@ public extension UIView {
         return UINib(nibName: name, bundle: bundle).instantiate(withOwner: nil, options: nil)[0] as? UIView
     }
 
+    /// SwifterSwift: Load view of a certain type from nib
+    ///
+    /// - Parameters:
+    ///   - withClass: UIView type.
+    ///   - bundle: bundle of nib (default is nil).
+    /// - Returns: UIView
+    class func loadFromNib<T: UIView>(withClass name: T.Type, bundle: Bundle? = nil) -> T {
+        let named = String(describing: name)
+        guard let view = UINib(nibName: named, bundle: bundle).instantiate(withOwner: nil, options: nil)[0] as? T else {
+            fatalError("First element in xib file \(named) is not of type \(named)")
```

Comment:
`fatalError` is so harsh and untestable here, what do you think of making the function return an optional view instead?

Reply:
There's a reason for this - it's intended to match `dequeueReusableCell`, which is also quite harsh. I almost always use optionals in my own code as well, but I think there are a few spots (like nib loading) where it's simply "developer error" if something goes wrong. UIKit will also throw an exception if a nib file is not found, for example. Reasonable behavior in my opinion. Not sure a user could reasonably handle the error and continue here.

Having said that, happy to change it if you feel otherwise. Users can always throw their own fatalErrors when calling this function.

Reply:
I used to do that in the past as well but the thing is discussing this one time, I've learned and it's been working for me since, that for loading static resources which should always be there unless someone deletes it does make sense to fail fatal or at least assert it, so we verify in development that it is something wrong with our asset, xib ... or any other static resource. To me, the ideal is always returning a non-optional(because the resource should be there) and having a `assert` that only triggers in debug builds to break when something wrong with it. With that, we achieve both safety when loading resources and avoid a harsh crash in a production application.
Something like this 
```swift
guard let view = UINib(nibName: named, bundle: bundle).instantiate(withOwner: nil, options: nil)[0] as? T else {
  assertionFailure("First element in xib file \(named) is not of type \(named)")// Assert so we are warned on development
  return T()  // Always return something because the static resource it is expected to be there always, but don't crash in production and avoid the nil checks around calls.
}
```
Those are my 2 cents :)

Reply:
Happy to make that change. @omaralbeik, what do you think?

Reply:
I see your point, let's go with the `fatalError` approach then 👍